### PR TITLE
Fix existing test suites given the introduction of the contract registry  

### PR DESCRIPTION
**Pull request by saturnial**
This PR fixes all of our existing tests given the introduction of the contract registry.

### CODE REVIEW

File: test/ts/migrations/3_deploy_test_contracts_test.ts [hunk 1]
```diff
@@ -14,116 +14,98 @@ const web3Utils = new Web3Utils(web3);
...
 // Import our set of test Contracts
-import { DummyContractContract } from "../../../types/generated/dummy_contract";
+import { DummyContractContract as DummyContract } from "../../../types/generated/dummy_contract";
 import { MockDebtRegistryContract } from "../../../types/generated/mock_debt_registry";
 import { MockERC20TokenContract } from "../../../types/generated/mock_e_r_c20_token";
 import { MockERC721TokenContract } from "../../../types/generated/mock_e_r_c721_token";
+import { MockERC721ReceiverContract } from "../../../types/generated/mock_e_r_c721_receiver";
 import { MockDebtTokenContract } from "../../../types/generated/mock_debt_token";
-import { MockTermsContractContract } from "../../../types/generated/mock_terms_contract";
+import { MockTermsContractContract as MockTermsContract } from "../../../types/generated/mock_terms_contract";
```
[the remark this answers is not in the document]
Comment (saturnial) [reply test/ts/migrations/3_deploy_test_contracts_test.ts]: Right? Read so much better.

File: test/ts/migrations/2_deploy_dharma_contracts_test.ts [hunk 2]
```diff
@@ -35,89 +41,192 @@ contract("Migration #2: Deploying Dharma Contracts", async (ACCOUNTS) => {
...
 
         it("lists the correct accounts as owner", async () => {
-            _.forEach(contractOwners, (value: any, i: number) => {
-                expect(wallet.isOwner.callAsync(value)).to.eventually.be.true;
-            });
+            await Promise.all(
+                contractOwners.map(async (owner) => {
+                    return expect(wallet.isOwner.callAsync(owner)).to.eventually.be.true;
+                }),
+            );
```
[the remark this answers is not in the document]
Comment (saturnial) [reply test/ts/migrations/2_deploy_dharma_contracts_test.ts]: Does that mean we want to add a method to the wallet that returns the number of owners?

[the remark this answers is not in the document]
Comment (nadavhollander) [reply test/ts/migrations/2_deploy_dharma_contracts_test.ts]: Don't think we necessarily need to do that -- can instead just use 

```javascript 
const owners = await wallet.owners.callAsync();
const numOwners = owners.length;
```

[the remark this answers is not in the document]
Comment (saturnial) [reply test/ts/migrations/2_deploy_dharma_contracts_test.ts]: Ah, there's a `getOwners` method available. I'll use that.

File: test/ts/migrations/2_deploy_dharma_contracts_test.ts [hunk 3]
```diff
@@ -35,89 +41,192 @@ contract("Migration #2: Deploying Dharma Contracts", async (ACCOUNTS) => {
...
         });
 
         it("lists the correct number of required authorizations", async () => {
             const requiredComputed = new BigNumber(Math.ceil(contractOwners.length / 2));
```
[the remark this answers is not in the document]
Comment (saturnial) [reply test/ts/migrations/2_deploy_dharma_contracts_test.ts]: 💯

**Pull request merged**